core/templates/dev/head/pages/exploration_player/LearnerLocalNav.js

```diff
@@ -59,22 +58,56 @@ oppia.controller('LearnerLocalNav', [
             };
 
             $scope.submitSuggestion = function() {
-              $uibModalInstance.close({
+              data = {
```

Comment:
You should do `var data = {` here if this is your first time defining the variable (similarly below).

Reply:
Done